.github/workflows/python-security.yml

```diff
@@ -0,0 +1,58 @@
+# Workflow regarding python security scanning.
+name: Python Security
+
+on:
+  push:
+    branches:
+      # Run on our main branch
+      - main
+    paths:
+      - example-python/**
+  pull_request:
+    # Run for any pull requests
+    paths:
+      - example-python/**
+jobs:
+  # https://github.com/marketplace/actions/anchore-container-scan
+  dependency-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4  # Checkout the current branch/merge state
+      - name: Grype Scan
```

Comment:
I don't understand where I can find the result of Grype Scan ?

Reply:
@leobr29 This is a good point, I didn't think about uploading a sarif for grype scan, but it seems it has the functionality for it so I may implement this now.

Currently the job will just fail if any vulnerabilities are found.

Reply:
@leobr29 So I have added the sarif upload for the grype report. I believe they will show up here if there are any vulnerabilities found : https://github.com/British-Oceanographic-Data-Centre/amrit-repos/security/dependabot

The only real way to test it would be to add a dependency that has a known vulnerability 🤔